examples/fine-tuning/osft/README.md

```diff
@@ -104,34 +112,35 @@ osft(..., use_processed_dataset=True)
 
 ### Setup Workbench
 
-* Access the OpenShift AI dashboard, for example from the top navigation bar menu:
+- Access the OpenShift AI dashboard, for example from the top navigation bar menu:
 ![](./docs/01.png)
-* Log in, then go to _Data Science Projects_ and create a project:
+- Log in, then go to _Data Science Projects_ and create a project:
 ![](./docs/02.png)
-* Once the project is created, click on _Create a workbench_:
+- Once the project is created, click on _Create a workbench_:
 ![](./docs/03.png)
-* Then create a workbench with the following settings:
-    * Select the `Jupyter | Minimal | CPU | Python 3.12` notebook image if you want to run CPU based evaluation, `Jupyter | Minimal | CUDA | Python 3.12` for NVIDIA GPUs evaluation and `Medium` container size:
+- Then create a workbench with the following settings:
+  - Select the `Jupyter | Minimal | CPU | Python 3.12` notebook image if you want to run CPU based evaluation, `Jupyter | Minimal | CUDA | Python 3.12` for NVIDIA GPUs evaluation and `Medium` container size:
     ![](./docs/04a.png)
-    * Add an accelerator if you plan on evaluating your model on GPUs (faster):
+  - Add an accelerator if you plan on evaluating your model on GPUs (faster):
     ![](./docs/04b.png)
         > [!NOTE]
         > Adding an accelerator is only needed to test the fine-tuned model from within the workbench so you can spare an accelerator if needed.
```

Comment:
@tarun-etikala - unrelated to changes in this PR but would you know why do some of the [!NOTE] render correctly and some come up raw?